Project.toml

```
@@ -3,8 +3,19 @@ uuid = "45b822f7-b0f9-4a40-9473-dffa0412f8e7"
 authors = ["willtebbutt <wt0881@my.bristol.ac.uk>"]
 version = "0.1.0"
 
+[deps]
+AbstractGPs = "99985d1d-32ba-4be9-9821-2ec096f28918"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+KernelFunctions = "ec8451be-7e33-11e9-00cf-bbf324bd1392"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
 [compat]
-julia = "1"
+AbstractGPs = "0.1.1"
+Distributions = "0.19, 0.20, 0.21, 0.22, 0.23"
+KernelFunctions = "0.4"
+julia = ">=1.3"
```

```suggestion
julia = "1.3"
```
This means the same thing :)